src/cli.ts

```diff
@@ -23,7 +24,7 @@ const cli = meow(`
 
 	Examples
 	  $ asyncapi context add dummy ./asyncapi.yml
-	  $ asyncapi validate --context=dummy
+      $ asyncapi validate --context=dummy
```

Comment:
We should also add example for the `file` flag.

```suggestion
        $ asyncapi validate --context=dummy
        $ asyncapi validate --file=./asyncapi.yml
```

Reply:
Could you align this line? :)